lib/cylc/cfgspec/globalcfg.py

```diff
@@ -30,6 +41,50 @@
 
 "Cylc site and user configuration file spec."
 
+interval_parser = DurationParser()
+
+def _coerce_interval( value, keys, args, back_comp_unit_factor=1 ):
```

Comment:
Aren't these the same as in `suite.py`? Could they just be renamed without the underscore and imported directly from there?